kong.conf.default

```diff
@@ -173,6 +173,15 @@
                                  #   It runs DB-less and receives configuration
                                  #   updates from a control plane node.
 
+#cluster_mtls = shared           #
+                                 # Valid values to this setting are:
+                                 #
+                                 # - `shared`: use a shared certificate/key
+                                 #   pair specified with the `cluster_cert`
+                                 #   and `cluster_cert_key` settings.
+                                 # - `pki`: use `cluster_ca_cert` and
+                                 #   `cluster_server_name` for verification.
+
 #cluster_cert =                  # Filename of the cluster certificate to use
```

Comment:
We should add for `cluster_cert` and `cluster_cert_key` that if `cluster_mtls = pki`, then this is the cluster CA issued DP cert presented to the CP during mTLS handshake and should be a different cert for each DP node.